### DISCUSSION TITLE
리스트의 글자크기를 바꿀 수 있는 기능은 없나요?

### DISCUSSION THREAD

**Discussion by Rdtddt** [pronist/hELLO#217] [Q&A] [answered]
안녕하세요, 좋은 스킨 정말 잘 쓰고 있습니다.
최근에 하나 문제가 생겨 문의드립니다.

저는 게시물을 쓸 때, 본문3(가장 작은 크기) 로 게시물을 쓰는데
리스트 기능(앞에 점 찍고 나열하는 기능) 을 넣으면
글자 크기가 다시 본문2 정도로 돌아옵니다.

리스트를 만들어 놓고 텍스트를 블럭 친 뒤 크기를 바꾸려고 해도
이 때는 '문단 모양'(글자 크기 바꾸는 곳) 칸이 회색으로 변하면서 아예 비활성화가 됩니다.

CSS에도 관련 부분을 찾아보려고 해도
리스트 관련 문단 모양을 만지는 곳이 안보이고

'본문2(0.9375)' 정도에 해당되는 숫자가 적힌 코드가 안보입니다.
리스트의 글자크기/간격 등을 바꿀 수 있는 방법이 없을까요?

**Comment by pronist** [comment, marked as answer]
@Rdtddt 안녕하세요.

본문에 대한 세세한 부분은 별도의 기능으로 제공하지 않습니다. CSS 에서 글 본문에 대한 내용은 `#content .contents_style` 를 비롯한 그 이하 자식 요소에 있으므로 그 부분을 위주로 수정하시면 될 것 같습니다. 리스트나 줄 간격 모두 마찬가지입니다.
